components/translate/core/browser/brave_translate_features.h

```diff
@@ -0,0 +1,37 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_TRANSLATE_CORE_BROWSER_BRAVE_TRANSLATE_FEATURES_H_
+#define BRAVE_COMPONENTS_TRANSLATE_CORE_BROWSER_BRAVE_TRANSLATE_FEATURES_H_
+
+#include "base/feature_list.h"
+#include "base/metrics/field_trial_params.h"
+
+namespace translate {
+
+extern const char kBraveTranslateOrigin[];
+extern const char kBraveTranslateScriptURL[];
```

Comment:
maybe move this to `translate_network_constants.h`?

Reply:
`translate_network_constants.h` is located inside `\chrome`

Reply:
right. then maybe it's better to create a constants file in `components`? doesn't look good to keep some arbitrary constants with features.

Reply:
Moved